content/blog/2017/2017-02-06-declarative-pipeline-ga.adoc

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Declarative Pipeline Syntax 1.0 is GA"
+tags:
+- pipeline
+- blueocean
+author: hrmpw
+---
+[NOTE]
+====
+This is a guest post by
+link:https://github.com/HRMPW[Patrick Wolf],
+Director of Product Management at
+link:http://cloudbees.com[CloudBees]
+and contributor to
+the Jenkins project.
+====
+
+I am very excited to announce the addition of
+link:https://plugins.jenkins.io/pipeline-model-definition[Declarative Pipeline syntax]
+1.0 to
+link:https://plugins.jenkins.io/workflow-aggregator[Jenkins Pipeline.]
+We think this new syntax will enable everyone involved in DevOps, regardless of expertise,
```

Comment:
It seems there are no examples of declarative syntax in a blog post about declarative syntax ;)

So I'd link "new syntax" to the new syntax documentation: https://jenkins.io/doc/book/pipeline/